src/servsock.js

```diff
@@ -0,0 +1,54 @@
+var fs = require('fs');
+var net = require('net');
+
+export default class ServiceSocket {
+
+    constructor() {
+        this.connections = {};
+    }
+
+    init(handler, socket){
+        this.handler = handler;
+        this.socket = socket;
+
+        fs.stat(this.socket, (err, stats)=>{
+            if (err) {
+                return this.openServiceSocket();
+            }
+            fs.unlink(this.socket, (err)=>{
+                if(err){
+                    console.error(err); process.exit(0);
+                }
+                return this.openServiceSocket();
+            });
+        });
+    }
+
+    openServiceSocket(){
+        this.server = net.createServer((stream)=>{
+            let self = Date.now();
```

Comment:
Throughout CyTube, `self` is traditionally used as a lexical closure around `this` for legacy ECMAScript 5 code where JavaScript would mangle `this` in callbacks/nested functions.  Using it as something unrelated here seems like a weird naming choice.